dashboard/admin.py

```diff
@@ -70,6 +70,7 @@ def has_add_permission(self, request):
 class CourseAdmin(admin.ModelAdmin):
     inlines = [CourseViewOptionInline, ]
     form = CourseForm
+    fields = ('canvas_id', 'name', 'term', 'date_start', 'date_end', 'show_grade_counts', 'show_grade_type', 'data_last_updated')
```

Comment:
I do not see this newly added variable, "fields", used in the code anywhere.

Reply:
This changes the order that the fields appear in in the admin form. If I don't specify this, then `canvas_id` gets pushed to the bottom when it is read only.

Reply:
I see. Maybe add a comment line for this?

Reply:
Do you think we need a comment? In my view, it's documented well in the Django docs: https://docs.djangoproject.com/en/4.1/ref/contrib/admin/#django.contrib.admin.ModelAdmin.fields

Reply:
I guess the readonly bit is not said explicitly. Just don't want to clutter the code too much with comments.

Reply:
OK. The Django doc is enough.